src/modules/user/application/use_cases/resend_verification_email_use_case.py

```
@@ -0,0 +1,100 @@
+
+        Args:
+            email: Email del usuario al que reenviar la verificación
+
+        Returns:
+            bool: True si el email se envió exitosamente
+
+        Raises:
+            ValueError: Si el email no existe o ya está verificado
+        """
+        if not email or email.strip() == "":
+            raise ValueError("El email es requerido")
```

Ruff (0.14.4)

Avoid specifying long messages outside the exception class

(TRY003)